master/docs/developer/stats-service.rst

```diff
@@ -13,16 +13,19 @@ Here is a diagram demonstrating the working of the stats service:
 Stats Service
 -------------
 
-.. py:class:: StatsService
+.. py:class:: buildbot.statistics.stats_service.StatsService
 
-   An instance of this class functions as a :class:`BuildbotService`. The instance of the running
-   service is initialized in the master configuration file (see :bb:cfg:`stats-service` for more).
+   An instance of this class functions as a :class:`BuildbotService`.
+   The instance of the running service is initialized in the master configuration file (see :bb:cfg:`stats-service` for more information).
    The running service is accessible everywhere in Buildbot via the :class:`BuildMaster`.
    The service is available at ``self.master.namedServices['<service-name>']``.
    It takes the following intialization arguments:
 
-   * ``storage_backends``: A list of storage backends. These are instance of subclasses of :class:`StatsStorageBase`.
-   * ``name``: The name of this service. This name can be used to access the running instance of this service using ``self.master.namedServices[name]``.
+   ``storage_backends``
+     A list of storage backends. These are instance of subclasses of :class:`StatsStorageBase`.
+   ``name``
```

Comment:
I did change the whole thing though (including the previously uploaded docs). I will see if anything remains that I haven't corrected.